python/format/scripts/validate.py

```diff
@@ -14,11 +17,16 @@
 
 FLAGS = flags.FLAGS
 
+
 def main(argv):
     del argv
     file = FLAGS.file
-    datasets.Dataset(file)
-    print("Done.")
+    try:
+        datasets.Dataset(file)
+        print("Done.")
```

Comment:
Minor point: wouldn't `logging.info` be better here?

Reply:
Done.